src/Apps/Artwork/Components/OtherWorks/ArtworkContexts/ArtworkGrids/ArtistArtworkGrid.tsx

```diff
@@ -45,6 +45,7 @@ export class ArtistArtworkGrid extends React.Component<ArtistArtworkGridProps> {
         <ArtworkGrid
           artworks={artist.artworks_connection}
           columnCount={[2, 3, 4]}
+          preloadImageCount={0}
```

Comment:
Should this be a default prop?

Reply:
This definitely shouldn't be! That'd mean they always lazy load by default... which, I mean might be good sometimes, but might not others.